code/modules/projectiles/guns/energy.dm

```diff
@@ -593,4 +548,346 @@
 	var/obj/screen/ammo/A = user.hud_used.ammo //The ammo HUD
 	A.update_hud(user)
 
-	return TRUE
+/obj/item/weapon/gun/energy/lasgun/lasrifle/update_item_state(mob/user) //Without this override icon states for wielded guns won't show. because lasgun overrides and this has no charge icons
+	item_state = "[initial(icon_state)][flags_item & WIELDED ? "_w" : ""]"
+
+
+//TE Tier 1 Series//
+
+//TE Standard Laser rifle
+
+/obj/item/weapon/gun/energy/lasgun/lasrifle/standard_marine_rifle
+	name = "\improper Terra Experimental laser rifle"
+	desc = "A TerraGov standard issue laser rifle otherwise known as TE-R abbreviated has an integrated charge selector for normal and high settings. Uses standard Terra Experimntal, TE abbreviated power cells."
```

Comment:
```suggestion
	desc = "A TerraGov standard issue laser rifle otherwise known as TE-R abbreviated has an integrated charge selector for normal and high settings. Uses standard Terra Experimental (abbreviated as TE) power cells."
```